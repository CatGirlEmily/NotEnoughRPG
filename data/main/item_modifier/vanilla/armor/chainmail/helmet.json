[
  {
    "function": "minecraft:set_components",
    "components": {
      "minecraft:attribute_modifiers": {
        "modifiers": [],
        "show_in_tooltip": true
      },
      "minecraft:custom_data": "{chainmail_helmet:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+12",
      " ",
      "§a§lUNCOMMON HELMET"
    ],
    "mode": "replace_all"
  }
]