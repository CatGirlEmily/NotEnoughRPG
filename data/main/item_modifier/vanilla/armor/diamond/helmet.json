[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{DIAMOND_HELMET:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+15",
      " ",
      "§a§lUNCOMMON HELMET"
    ],
    "mode": "replace_all"
  }
]