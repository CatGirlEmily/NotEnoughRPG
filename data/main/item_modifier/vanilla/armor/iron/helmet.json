[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{IRON_HELMET:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+12",
      " ",
      "§f§lCOMMON HELMET"
    ],
    "mode": "replace_all"
  }
]