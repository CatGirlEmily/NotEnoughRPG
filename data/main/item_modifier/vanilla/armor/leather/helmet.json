[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{LEATHER_HELMET:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+5",
      " ",
      "§f§lCOMMON HELMET"
    ],
    "mode": "replace_all"
  }
]