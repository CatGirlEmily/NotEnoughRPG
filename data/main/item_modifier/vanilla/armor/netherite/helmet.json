[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{NETHERITE_HELMET:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+25",
      " ",
      "§5§lEPIC HELMET"
    ],
    "mode": "replace_all"
  }
]