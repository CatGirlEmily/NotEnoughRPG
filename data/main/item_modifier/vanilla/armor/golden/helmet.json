[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{GOLDEN_HELMET:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+10",
      " ",
      "§f§lCOMMON HELMET"
    ],
    "mode": "replace_all"
  }
]